client/sandbox/react-nextjs/app/play/check-admin-cache/page.tsx

```diff
@@ -1,12 +1,14 @@
 import { init } from '@instantdb/admin';
-const APP_ID = '137ace7a-efdd-490f-b0dc-a3c73a14f892';
-const ADMIN_TOKEN = '82900c15-faac-495b-b385-9f9e7743b629';
+
+import config from '../../../config';
+
 const db = init({
-  appId: APP_ID,
-  adminToken: ADMIN_TOKEN,
-  apiURI: 'http://localhost:8888',
+  ...config,
+  adminToken: process.env.INSTANT_ADMIN_TOKEN!,
 });
 
+export const dynamic = 'force-dynamic';
```

Comment:
I had to add this to get the build to pass. I _think_ this may be because nextjs 15 handles the fetch differently, when the explict no-store is not set. It _still_ makes the request, but just doesn't cache anything. 

This breaks our build, as the `db` in this case is not real, and the fetch request fails